bin/config.js

```diff
@@ -36,46 +28,41 @@ const prepareIcons = (input) => {
   return files
 }
 
-const prepareOutput = (output) => {
-  const result = {
-    ...defaultConfig.output,
-    icons: { ...defaultConfig.output.icons },
-  }
-
-  if (typeof output == 'string') result.path = output
-  else {
-    if (output.path) result.path = output.path
-    if (output.publicPath) result.publicPath = output.publicPath
-
-    if (typeof output.icons == 'boolean')
-      result.icons = { ...defaultConfig.output.icons }
-    else if (typeof output.icons == 'string') {
-      result.icons = {
-        files: /.svg$/.test(output.icons) ? null : output.icons,
-        sprite: appendExtension(output.icons, '.svg'),
-      }
-    } else result.icons = { ...output.icons }
-
-    if (result.icons.files)
-      result.icons.files = path.resolve(result.path, result.icons.files)
-    if (result.icons.sprite)
-      result.icons.sprite = path.resolve(result.path, result.icons.sprite)
-  }
-
-  return result
+const mergeAndPrepareConfigs = (...configs) => {
+  // Create absolute path for icons so they can be merged
+  configs.forEach(({ config }) => {
+    if ((config || {}).icons) config.icons = prepareIcons(config.icons)
```

Comment:
I cannot find this function.

Reply:
You mean `prepareIcons`?

Reply:
Yes

Reply:
it's 15 lines up

Reply:
Sorry, I thought it's implemented by yourself but it was already there.